integrations/ros.rst

```diff
@@ -0,0 +1,211 @@
+.. _integrations_ros:
+
+|ros_logo| ROS
+==============
+
+.. include:: ../common/experimental_warning.inc
+
+Conan provides integration for your Robot Operating System (ROS) C/C++ based projects.
+This will allow you to consume Conan packages inside your ROS package projects.
+The Conan packages can be installed and used in CMake with the help of the :ref:`ROSEnv generator <conan_tools_ros_rosenv>` created
+for the purpose.
+
+It provides a clean integration that requires no Conan-specific modifications in your *CMakeLists.txt*.
+
+.. note::
```

Comment:
I think it would be nice to add some pre-requisites to run the example, like for example run everything inside the `osrf/ros:humble-desktop` docker image to test it.

Reply:
done